File: 0412-fizz-buzz/0412-fizz-buzz.java
class Solution {
    public List<String> fizzBuzz(int n) {
        List<String> answer= new ArrayList<String>();

        for(int i=1;i<=n;i++){
            String t;
            if(i%15==0)
                answer.add("FizzBuzz");
            else if(i%3==0)
                answer.add("Fizz");
            else if(i%5==0)
                answer.add("Buzz");
            else
                answer.add(i+"");
        }

        return answer;
    }
}

File: 0412-fizz-buzz/0412-fizz-buzz.py
class Solution:
    def fizzBuzz(self, n: int) -> List[str]:
        
        answer = [] 
        for i in range(1,n+1):
            t = str(i)
            if i%15==0:
                t = "FizzBuzz"
            elif i%5==0:
                t = "Buzz"
            elif i%3==0:
                t = "Fizz"
            answer.append(str(t))
        
        return answer
        